src/Models/WorkflowJob.php

```diff
@@ -170,6 +172,20 @@ public function retry(): void
             throw new CannotRetryJobException();
         }
 
+        if (config('queue.failed.driver') == 'database-uuids') {
+            $job = DB::connection(config('queue.failed.database'))
+                ->table(config('queue.failed.table'))
+                ->select('uuid')
+                ->where('payload', 'LIKE', "%{$this->uuid}%")
+                ->first();
+            if ($job) {
+                Artisan::call('queue:retry', [
+                    'id' => $job->uuid
+                ]);
+                return;
+            }
+        }
```

Comment:
I'm not sure on this approach. I think instead of calling `artisan queue:retry` here, it may be simpler/easier to delete/prune the failed job instead before dispatching. Though that of course has the caveat of the job having a different UUID than the one that was stored in the `failed_jobs` table. On the flip side, that is the existing behaviour, so it may be best to keep it that way until a next major version. Thoughts?